benchmarks/scripts/compare_benchmark_results.sh

```
@@ -0,0 +1,102 @@
+#!/usr/bin/env zsh
```

Shellcheck (0.11.0)

[error] ShellCheck only supports sh/bash/dash/ksh/'busybox sh' scripts. Sorry!

(SC1071)